Shared/Views/ShareCampaignView.swift

```diff
@@ -80,7 +81,7 @@ struct ShareCampaignView: View {
     var body: some View {
         NavigationView {
             Form {
-                Section(header: headerView.textCase(nil)) {
+                Section(header: headerView.textCase(nil), footer: Text("Some popular social media platforms such as Discord may not display rounded corners as intended.")) {
```

Comment:
Shall we only display the note when clipCorners is true?

Reply:
done